igor-web/src/main/groovy/com/netflix/spinnaker/igor/travis/service/TravisService.groovy

```diff
@@ -130,24 +151,32 @@ class TravisService implements BuildService {
 
     Commit getCommit(String repoSlug, int buildNumber) {
         Builds builds = getBuilds(repoSlug, buildNumber)
-        if (builds.commits) {
+        if (builds?.commits) {
             return builds.commits.first()
         }
         throw new NoSuchFieldException("No commit found for ${repoSlug}:${buildNumber}")
     }
 
 
     List<Repo> getReposForAccounts() {
-        log.debug "fetching repos for relevant accounts only"
-        List<Repo> repos = []
-        getAccounts().accounts.each { account ->
-            Repos accountRepos = travisClient.repos(getAccessToken(), account.login)
-            accountRepos.repos.each { repo ->
-                log.debug "[${account.login}] [${repo.slug}]"
+        new SimpleHystrixCommand<List<Repo>>(
+            groupKey, buildCommandKey("getReposForAccounts"),
+            {
+                log.debug "fetching repos for relevant accounts only"
+                List<Repo> repos = []
+                getAccounts().accounts.each { account ->
+                    Repos accountRepos = travisClient.repos(getAccessToken(), account.login)
+                    accountRepos.repos.each { repo ->
+                        log.debug "[${account.login}] [${repo.slug}]"
+                    }
+                    repos.addAll accountRepos.repos
+                }
+                return repos
+            },
+            {
+                return new ArrayList<Repo>()
```

Comment:
ie https://github.com/spinnaker/igor/blob/master/igor-web/src/main/groovy/com/netflix/spinnaker/igor/travis/TravisBuildMonitor.groovy#L159

I think it is best to return an empty list, because the list is reused several places in the TravisBuildMonitor